src/index.html

```diff
@@ -11,67 +11,68 @@
       href="https://fonts.gstatic.com"
       crossorigin="use-credentials"
     >
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+    <link
+     href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
 
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="#">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="#" class="nav__link">Apple</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+            <a href="#" class="nav__link">Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Смарфоны и телефоны</a>
+            <a href="#" class="nav__link">Смарфоны и телефоны</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Ноутбуки и компьютеры</a>
+            <a href="#" class="nav__link">Ноутбуки и компьютеры</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Гаджеты</a>
+            <a href="#" class="nav__link">Гаджеты</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Планшеты</a>
+            <a href="#" class="nav__link">Планшеты</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Фото</a>
+            <a href="#" class="nav__link">Фото</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Видео</a>
+            <a href="#" class="nav__link">Видео</a>
           </li>
         </ul>
       </nav>
     </header>
 
     <main class="catalog">
       <div class="catalog__card">
-        Product card
+      Product card
       </div>
       <div class="catalog__card">
-        Product card
+      Product card
       </div>
       <div class="catalog__card">
-        Product card
+      Product card
       </div>
       <div class="catalog__card">
-        Product card
+      Product card
       </div>
       <div class="catalog__card">
-        Product card
+      Product card
       </div>
       <div class="catalog__card">
-        Product card
+      Product card
       </div>
       <div class="catalog__card">
-        Product card
+      Product card
```

Comment:
Why did you delete spaces here? They're needed 🥲

Reply:
Add spaces back)

Reply:
Sorry! I`ve forgotten make 'git push origin develop'.

Reply:
Sorry! I`ve forgotten make 'git push origin develop'.